Documents/Backend_documentation/swagger.yaml

```diff
@@ -109,17 +103,14 @@ paths:
               $ref: "#/components/schemas/user"
         "404":
           description: returns a 404 user not found response code
+        "401":
+          description: returns 401 unauthorised request when JWT token can't be decoded or issuer is invalid or token is expired
         "500":
           description: returns a 500 internal server error, the body contains the specific error
-    /api/user/{id}/{resource}:
+    /api/user/{resource}:
```

Comment:
What are the different resources? like applications/favorites ? Can you send in a list of the resources?